modules/nf-core/nanocomp/main.nf

```diff
@@ -0,0 +1,35 @@
+process NANOCOMP {
+    label 'process_medium'
+
+    conda "bioconda:nanocomp=1.21.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/nanocomp:1.21.0--pyhdfd78af_0':
+        'quay.io/biocontainers/nanocomp:1.21.0--pyhdfd78af_0' }"
+
+    input:
+    val filetype
+    path filelist
```

Comment:
```suggestion
    tuple val(meta), path(filelist)
```

We add meta to all main inputs to be able to pass additional information to the module